controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -117,6 +125,16 @@ protected void startUp() throws Exception {
                 eventProcessorsOpt.get().handleOrphanedReaders(processes);
             }
 
+            if (txnSweeperOpt.isPresent()) {
+                // Await initialization of transactionTasksOpt.
+                log.info("Awaiting StreamTransactionMetadataTasks to get ready");
+                txnSweeperOpt.get().awaitInitialization();
```

Comment:
Will explicitly add `throws InterruptedException` even though, `throws Exception` is currently present until we modify cluster's addListener interface.